KlingonAssistant/data/mem-14-r.xml

```diff
@@ -543,11 +543,11 @@ According to a message on the {s.k:src} newsgroup, the first element of this wor
       <column name="entry_name">raw</column>
       <column name="part_of_speech">v:t_c</column>
       <column name="definition">lightning</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">blitzen</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{pe'bIl:n}, {tuD:v}, {jev:v:1}</column>
-      <column name="notes"></column>
+      <column name="notes">Although glossed as a noun, all weather terms are verbs, so this is the verb meaning "it is lightning".</column>
```

Comment:
I'm not happy either, but actually, I'm not happy with the original message from Maltz. "tornado" is not english either, so the user may think it's a mistake in the entry. Maybe just change this to 
"Although glossed as a noun, this weather terms is a verb, meaning "it is lightning"."